### PR TITLE
ED-2086: OCI support for Deploy/Lern/LernFlinkJobs - Superseeds PR#130

### PR DESCRIPTION
**Pull request by ddevadat**
Added OCI support for Deploy/Lern/LernFlinkJobs

This PR superseeds https://github.com/Sunbird-Lern/data-pipeline/pull/130

### CODE REVIEW

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 [hunk 1]
```diff
@@ -4,8 +4,8 @@ dockerhub: {{ dockerhub }}
 repository: {{flink_repository|default('data-pipeline')}}
 image_tag: {{ image_tag }}
 checkpoint_store_type: {{ checkpoint_store_type }}
-cloud_storage_key: {{ cloud_private_storage_accountname }}
-cloud_storage_secret: {{ cloud_private_storage_secret }}
+cloud_storage_key: {{ cloud_storage_key }}
```
Comment (AmiableAnil) [inline kubernetes/helm_charts/datapipeline_jobs/values.j2:7]: Have you added `cloud_storage_key` with same as `cloud_private_storage_accountname` if values are same please keep the `cloud_private_storage_accountname`. Same for `cloud_private_storage_secret`

Comment (ddevadat) [reply kubernetes/helm_charts/datapipeline_jobs/values.j2:7]: done

**Comment by AmiableAnil** [comment]
@ddevadat Please raise this PR against release-5.0.2

**Comment by ddevadat** [comment]
closed in favour of PR https://github.com/Sunbird-Lern/data-pipeline/pull/146

**Pull request closed without merging**